src/axom/slic/interface/slic.hpp

```diff
@@ -168,6 +168,13 @@ void addStreamToAllMsgLevels(LogStream* ls);
  * \param [in] filter_duplicates optional parameter that indicates whether
  * duplicate messages resulting from running in parallel will be filtered out.
  * Default is false.
+ * \note When used within an MPI distributed environment, and:
```

Comment:
Same comments that I made earlier about grouping the methods with Doxygen syntax and preface them with a single comment about their collective nature.